docs/src/markdown/extensions/snippets.md

```diff
@@ -237,6 +237,19 @@ If either of these is set to zero, the limits will be ignored.
 
 To pass arbitrary HTTP headers in every HTTP request use `url_request_headers`.
 
+/// new | New 10.16
```

Comment:
I usually use these "New" blocks just to inform the user when a new feature was added. You should describe the settings outside of the block where we talk about `url_request_headers`, `url_max_size`, and `url_timeout`. Eventually, I'll remove the really old "New" blocks, but the the documentation of the options should remain.